src/main/java/org/apache/maven/plugins/dependency/analyze/AbstractAnalyzeMojo.java

```diff
@@ -438,6 +478,40 @@ private boolean checkDependencies()
             warning = true;
         }
 
+        // log dependencies that weren't included
+        if ( verbose && !notIncludedUsedDeclared.isEmpty() )
```

Comment:
why are these logged? I thought the point of this PR was to avoid reporting on these.

Reply:
These are logged in the verbose section only. In the same way that the [existing verbose section](https://github.com/apache/maven-dependency-plugin/blob/master/src/main/java/org/apache/maven/plugins/dependency/analyze/AbstractAnalyzeMojo.java#L403) for the explicitly ignored dependencies, I think you'd also want to understand what was been ignore implicitly by not being covered within the include block.